core/src/main/scala/org/genivi/sota/core/transfer/DeviceUpdates.scala

```diff
@@ -98,24 +98,28 @@ object DeviceUpdates {
     }
   }
 
-  /**
-    * Find the [[UpdateRequest]]-s for:
-    * <ul>
-    *   <li>the given device</li>
-    *   <li>and for which [[UpdateSpec]]-s exist with Pending or InFlight [[UpdateStatus]]</li>
-    * </ul>
-    */
```

Comment:
because nobody updates them and what is written in the comment can easily be read in the code.